src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java

```diff
@@ -107,13 +108,14 @@ private void logConnectionStats(final Session session) {
             final Session.State state = session.getState();
 			final Configuration configuration = session.getCluster().getConfiguration();
 			final PoolingOptions poolingOptions = configuration.getPoolingOptions();
-			for (Host host : state.getConnectedHosts()) {
+			final Collection<Host> connectedHosts = state.getConnectedHosts();
+			for (Host host : connectedHosts) {
 				final HostDistance distance = configuration.getPolicies().getLoadBalancingPolicy().distance(host);
                 final int connections = state.getOpenConnections(host);
                 final int inFlightQueries = state.getInFlightQueries(host);
 				final int maxRequestsPerConnection = poolingOptions.getMaxRequestsPerConnection(distance);
-                logger.debug("connection_stats: host={} connections={}, current_load={}, max_load={}",
-                        host, connections, inFlightQueries, connections * maxRequestsPerConnection);
+                logger.debug("connection_stats: hosts_count={} host={} distance = {} connections={}, current_load={}, max_load={}",
```

Comment:
please drop the spaces around the `=` after `distance`

Reply:
sure, it was done accidentally. Thanks